src/mdbExtensionController.ts

```diff
@@ -89,14 +115,32 @@ export default class MDBExtensionController implements vscode.Disposable {
     let mdbConnectionString;
     if (this._connectionController) {
       const activeConnectionConfig = this._connectionController.getActiveConnectionConfig();
-      mdbConnectionString = activeConnectionConfig ? activeConnectionConfig.driverUrl : '';
+      mdbConnectionString = activeConnectionConfig
+        ? activeConnectionConfig.driverUrl
+        : '';
     }
-    const mongoDBShell = vscode.window.createTerminal({ name: 'MongoDB Shell', env: { MDB_CONNECTION_STRING: mdbConnectionString } });
+    const mongoDBShell = vscode.window.createTerminal({
+      name: 'MongoDB Shell',
+      env: { MDB_CONNECTION_STRING: mdbConnectionString }
+    });
     const shellCommand = vscode.workspace.getConfiguration('mdb').get('shell');
-    mongoDBShell.sendText(`${shellCommand} $MDB_CONNECTION_STRING; unset MDB_CONNECTION_STRING`);
+    mongoDBShell.sendText(
+      `${shellCommand} $MDB_CONNECTION_STRING; unset MDB_CONNECTION_STRING`
+    );
     mongoDBShell.show();
   }
 
+  public createPlayground(): void {
+    vscode.workspace
+      .openTextDocument({
+        language: 'mongodb',
+        content: '// The MongoDB playground'
```

Comment:
Max will think about what is better to auto-populate here. See: https://jira.mongodb.org/projects/VSCODE/issues/VSCODE-51